code/game/objects/items/weapons/grenades/flashbang.dm

```diff
@@ -5,39 +5,42 @@
 	origin_tech = "materials=2;combat=3"
 	light_power = 10
 	light_color = LIGHT_COLOR_WHITE
-	var/light_time = 2
-	var/range = 7
+
+	var/light_time = 0.2 SECONDS // The duration the area is illuminated
+	var/range = 7 // The range in tiles of the flashbang
 
 /obj/item/grenade/flashbang/prime()
 	update_mob()
-	var/flashbang_turf = get_turf(src)
-	if(!flashbang_turf)
+	var/turf/T = get_turf(src)
+	if(!T)
 		return
 
-	set_light(7)
-
+	// VFX and SFX
 	do_sparks(rand(5, 9), FALSE, src)
-	playsound(flashbang_turf, 'sound/effects/bang.ogg', 25, 1)
-	bang(flashbang_turf, src, range)
+	playsound(T, 'sound/effects/bang.ogg', 100, TRUE)
```

Comment:
I feel that flashbangs are already loud enough, though that might just be me

Reply:
From my testing it wasn't really any louder so 🤷‍♀️

Reply:
I ask the question "why bother with the increase if it wasnt louder"